src/components/Table/Budget/BudgetTargetTable/BudgetTargetTableView.tsx

```diff
@@ -125,7 +125,7 @@ function BudgetTargetView({
 	removeFilterListElements: (key: string, index?: number | undefined) => void;
 	currency: string;
 }) {
-	tableHeadings[5].label = "Total Amount " + `(${currency})`;
+	tableHeadings[5].label = `Total Amount (${currency})`;
```

Comment:
What is `5`?

Reply:
We have kept all of the table headings in seprate file so it can be used both in testing and component